swap_meet/decor.py

```diff
@@ -1,2 +1,8 @@
-class Decor:
-    pass
+from swap_meet.item import Item
+
+class Decor(Item):
+    def __init__(self, category="Decor", age=-1, condition=0):
+        super().__init__(category, age, condition)
```

Comment:
same thing here! Because we know this child class is called Decor and should have "decor" as its category, let's hardcode it in!